plutus-extra/src/Plutus/Contract/Test/Extra.hs

```diff
@@ -1,28 +1,65 @@
+{-# LANGUAGE PolyKinds #-}
 {-# LANGUAGE RankNTypes #-}
 
-module Plutus.Contract.Test.Extra (walletFundsChangeWithAccumState, walletFundsExactChangeWithAccumState) where
+module Plutus.Contract.Test.Extra (
+  walletFundsChangeWithAccumState,
+  walletFundsExactChangeWithAccumState,
+  valueAtAddressComputedFromAccumState,
+  dataAtAddressComputedFromAccumState,
+) where
+
+--------------------------------------------------------------------------------
 
 import Control.Foldl qualified as L
 
-import Control.Lens (at, (^.))
+--------------------------------------------------------------------------------
+
+import Control.Arrow ((>>>))
+import Control.Lens (at, view, (^.))
 import Control.Monad (unless)
+import Control.Monad.Freer (Eff, Member)
+import Control.Monad.Freer.Error (Error)
 import Control.Monad.Freer.Reader (ask)
-import Control.Monad.Freer.Writer (tell)
+import Control.Monad.Freer.Writer (Writer, tell)
 import Data.Foldable (fold)
+import Data.Kind (Type)
+import Data.List (foldl')
+import Data.Map qualified as Map
+import Data.Maybe (mapMaybe)
+import Data.Row (Row)
 import Data.Text.Prettyprint.Doc (Doc, colon, indent, pretty, viaShow, vsep, (<+>))
 import Data.Void
```

Comment:
Please give an explicit import list, [as required by the style guide](https://github.com/mlabs-haskell/styleguide#modules).

Reply:
`Data.Void` is importing `Void` correctly. I haven't done an explicit `Prelude` import list, assuming it's OK not to do so.

Reply:
Indeed.